src/ReleaseHistory.md

```diff
@@ -185,23 +185,23 @@
 * Multitool improvements--added "rebaseUri" and "absoluteUri" tasks, which will either make the URIs in a SARIF log relative to some base URI, or take base URIs stored in SARIF and make the URIs absolute again.
 * Added a "processing pipeline" model to the SARIF SDK in order to allow easy chaining of operations on SARIF logs (like making all URIs relative/absolute).
 
-## **v2.0.0-csd.1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.1)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.1))
```

Comment:
>1)) [](start = 221, length = 3)

Scrolling through this I noticed a bunch of spurious trailing parentheses.

Reply:
thanks! :)

---
In reply to: [290505867](https://github.com/microsoft/sarif-sdk/pull/1519#discussion_r290505867) [](ancestors = 290505867)